backend/networkTree.js

```diff
@@ -0,0 +1,149 @@
+class Queue {
+
+    constructor() { this.list = []; }
+
+    isEmpty() { return this.list.length === 0; }
+    enqueue(elem) { this.list.push(elem); }
+    dequeue() { if(!this.isEmpty()) { return this.list.shift(); } }
+}
+
+class networkNode {
```

Comment:
NetworkNode. We use PascalCase for class names.

Reply:
Also for the file name.

Reply:
Meant to fix it earlier, but totally forgot. Addressing it now